### DISCUSSION TITLE
v.0.19 - spectrum mode - please clarify wiki description of key [2]/[8]

### DISCUSSION THREAD

**Discussion by iu1mrd** [egzumer/uv-k5-firmware-custom#201] [Q&A] [answered]
Spectrum X-axis is ruled by values set by 
key [4] {16-32-64-128} 
and
key [1]/[7] {0.01k ... 100k}
let's say for example we have 128 x 100k = 12800k and this is displayed in middle low part as +/-6400k
So far so good
Now I push [8] and 6400k became 6000k ... I cant understand what this mean... can you help please?

in particular can you explain in details
" ... [ 1/7 ] - increases/decreases the...separation of a single bar in the graph ..."
versus
"...[ 2/8 ] - increases/decreases frequency step size ..." ? 

tnx
gio

**Comment by egzumer** [comment, marked as answer]
2/8 changes a value by which the graph is scrolled with up/down buttons
1/7 changes frequency step of consecutive bars

**Comment by ErikS-web** [comment]
As mentioned in the **_Online_ Manual.**.. our [Wiki](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Spectrum-analyzer)

**Comment by iu1mrd** [comment]
Thank you very much.
In other words: 1 / 7 and  2 / 8 act on two different and totally
independent parameters.
I was in doubt just because both have effect on one display "field".
Now clear.
